languages/ocaml.scm

```diff
@@ -473,6 +473,25 @@
   (_)
 )
 
+; Keep spacing around the list constructor if it is preceded by
+; a labeled argument, avoiding syntax errors.
+;
+; This is syntactically correct:
+;   foo ~arg :: []
+; This is not:
+;   foo ~arg::[]
+;
+; While both are correct if the argument is not labeled
+(
+  (application_expression
+    (_)*
+    .
```

Comment:
Aren't these two lines redundant?

Reply:
They are indeed ; it's fixed in the new version